tests/feeds/test_eur_usd_feed.py

```diff
@@ -0,0 +1,24 @@
+"""
+EUR/USD DataFeed Tests
+"""
+import pytest
+
+from telliot_feeds.feeds.eur_usd_feed import eur_usd_median_feed
+
+
+@pytest.mark.asyncio
+async def test_fetch_price():
+    """Fetch latest EUR/USD price"""
+    (value, _) = await eur_usd_median_feed.source.fetch_new_datapoint()
+    assert value > 0
+    print(value)
+
+
+def test_query_info():
```

Comment:
You can remove this test. `SpotPrice` query generation of its `queryId` & `queryData` has been tested. If you want to include the `queryId` & `queryData` somewhere in the file w/ the datafeed (as a docstring or comment), then that's an option